### ISSUE TITLE
Empty button should not flag if hidden from screen readers

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1717]
Example:
```
<div role="button" style="width: 26px; height: 37px; overflow: hidden; position: absolute; left: 38px; top: -2px; z-index: 35;" tabindex="-1" aria-hidden="true"><img alt="" src="https://maps.gstatic.com/mapfiles/transparent.png" draggable="false" usemap="#gmimap16" style="width: 26px; height: 37px; user-select: none; border: 0px; padding: 0px; margin: 0px; max-width: none;"><map name="gmimap16" id="gmimap16"><area log="miw" coords="13,0,4,3.5,0,12,2.75,21,13,37,23.5,21,26,12,22,3.5" shape="poly" tabindex="-1" title="" style="display: inline; position: absolute; left: 0px; top: 0px; cursor: pointer; touch-action: none;" aria-hidden="true"></map></div>
```

Because this has `tabindex="-1"` and `aria-hidden="true"` it shouldn't matter that there is no name on the button. However, this currently still flags an error:

<img width="1531" height="267" alt="Image" src="https://github.com/user-attachments/assets/2ff0fd5a-64dd-41a9-b2a1-563605810081" />
